src/utils/network/session-core.js

```diff
@@ -91,14 +90,13 @@ export const setDocContainerAclPermission = async (session, permissions, otherPo
   const containerUrl = getContainerUrl(session, 'Documents', INTERACTION_TYPES.SELF);
   const urlsToSet = [
     containerUrl,
-    `${containerUrl}Bank%20Statement/`,
-    `${containerUrl}Passport/`,
-    `${containerUrl}Drivers%20License/`
+    `${containerUrl}PASS_Bank_Statement/`,
```

Comment:
Wouldn't moving these all into a `PASS` container have the same effect?
e.g. `kahung.solidcommunity.net/pod/PASS/Bank_Statement` , `kahung.solidcommunity.net/pod/PASS/Passport`

Reply:
It would actually. I wasn't sure if that's something we wanted at the moment, so I've just renamed them for now with just prefixes.

Reply:
Should be an easy swap anyways, might as well try that for this update. 👍

Reply:
Updated containers to be stored within a PASS container